Baikal/Utils/cl_program.h

```diff
@@ -95,5 +96,7 @@ namespace Baikal
         CLWContext m_context;
         std::set<std::string> m_included_headers; ///< Set of included headers
 
+        static std::unordered_map<std::string, std::shared_ptr<std::mutex>> s_binary_cache_names;
```

Comment:
It's not a part of the class interface, so it might be moved into the .cpp file.

Reply:
Moved into cl_program.cpp